### PR TITLE
keep: create webiny-js

### PR DESCRIPTION
**Pull request by Shipow**
HUB generated configuration

### CODE REVIEW

File: configs/webiny-js.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "https://docs.webiny.com/sitemap.xml"
+  ],
+  "sitemap_alternate_links": true,
+  "custom_settings": {
+    "attributesForFaceting": [
+      "language",
+      "version"
+    ]
+  },
+  "min_indexed_level": 0,
```
Comment (s-pace) [inline configs/webiny-js.json]: Can we make sure to not define this value to 0 since it is its default value?

File: configs/webiny-js.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "stop_urls": [],
+  "selectors": {
+    "text": "[class^='docItemContainer_'] p, [class^='docItemContainer_'] li",
+    "lvl1": "[class^='docItemContainer_'] h1",
+    "lvl2": "[class^='docItemContainer_'] h2",
+    "lvl3": "[class^='docItemContainer_'] h3",
+    "lvl4": "[class^='docItemContainer_'] h4",
+    "lvl5": "[class^='docItemContainer_'] h5",
+    "lvl6": "[class^='docItemContainer_'] h6",
+    "lvl0": {
```
Comment (s-pace) [inline configs/webiny-js.json]: Can we make sure to order correctly the lvl in an increasing order please?

File: configs/webiny-js.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      "selector": ".menu__link--sublist.menu__link--active",
+      "global": true,
+      "default_value": "Documentation"
+    }
+  },
+  "selectors_exclude": [
+    ".hash-link"
+  ],
+  "sitemap_urls": [
```
Comment (s-pace) [inline configs/webiny-js.json]: Should be defined after `start_urls `

File: configs/webiny-js.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "https://docs.webiny.com/sitemap.xml"
+  ],
+  "sitemap_alternate_links": true,
```
Comment (s-pace) [inline configs/webiny-js.json]: Should be defined after `sitemap_urls`

Comment (Shipow) [reply configs/webiny-js.json]: before ?

File: configs/webiny-js.json (hunk 1 above)
Comment (Shipow) [reply configs/webiny-js.json]: It's your own code, I didn't change the class function. Please PR it as this should be easier.

Comment (Shipow) [reply configs/webiny-js.json]: should we stop generating the demo meanwhile?

Comment (s-pace) [reply configs/webiny-js.json]: Will do, let's create some this way, I will fix them up later on then.

File: configs/webiny-js.json (hunk 4 above)
Comment (s-pace) [reply configs/webiny-js.json]: No afterward

**Pull request merged**